app/src/sharedTest/java/org/oppia/app/option/OptionsFragmentTest.kt

```diff
@@ -0,0 +1,119 @@
+package org.oppia.app.option
+
+import android.view.View
+import android.widget.SeekBar
+import androidx.recyclerview.widget.RecyclerView
+import androidx.test.core.app.ActivityScenario
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.ViewAction
+import androidx.test.espresso.action.CoordinatesProvider
+import androidx.test.espresso.action.GeneralClickAction
+import androidx.test.espresso.action.Press
+import androidx.test.espresso.action.Tap
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.contrib.RecyclerViewActions
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.rule.ActivityTestRule
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+
+/** Tests for [OptionsFragment]. */
+@RunWith(AndroidJUnit4::class)
+class OptionsFragmentTest {
```

Comment:
These test cases test only the functionalities. We can also add test cases to check that the content on the OptionsActivity and its internal activities is displayed correctly. 

Also, for example, we select large font and come back to OptionsActivity and open StoryTextSize then we can check that it has selected `Large` button by default. Similarly for other two activities.